app/assets/javascripts/components/activity/activity_table_row.jsx

```diff
@@ -65,7 +65,9 @@ const ActivityTableRow = React.createClass({
 
     let diffViewer;
     if (this.props.revision && this.props.revision.api_url) {
-      diffViewer = <DiffViewer revision={this.props.revision} />;
+      diffViewer = <DiffViewer revision={this.props.revision}
```

Comment:
I am showing the cum count for the range of revs.  I could calculate the cum char change by looking at the `size` attribute of each rev - but since that is already calculated by an API call, and is available in the parent component, I thought it would be better to just add the props var.  

Or, if the story does not require it, we could just show the char count only for the single-rev case.

Thoughts?